tests/__init__.py

```diff
@@ -0,0 +1,40 @@
+# __init__.py ---
+#
+# Filename: __init__.py
+# Description:
+# Author: Elric Milon
+# Maintainer:
+# Created: Fri Jun 10 10:26:11 2016 (+0200)
+
+# Commentary:
+#
+#
+#
+#
+
+# Change Log:
+#
+#
+#
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or (at
+# your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with GNU Emacs.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+
+# Code:
+
+adfadsf
```

Comment:
That's on the DROPME commit, it's used to make the current PR tester fail right away and not clog the job queue with useless jobs while I fix/improve the pipeline